src/main/java/edu/harvard/iq/dataverse/api/Pids.java

```diff
@@ -45,7 +45,7 @@ public Response getPid(@QueryParam("persistentId") String persistentId) {
         } catch (WrappedResponse ex) {
             return error(Response.Status.FORBIDDEN, BundleUtil.getStringFromBundle("api.errors.invalidApiToken"));
         }
-        String baseUrl = System.getProperty("doi.baseurlstringnext");
+        String baseUrl = systemConfig.getDataCiteRestApiUrlString();
```

Comment:
So this API endpoint only supports DataCite?

Reply:
Good catch. Yes, and I think the docs say that. However, the other places that were DataCite-specific had been resolved in earlier PRs that were merged into dev. I just resolved the conflicts in this PR and you can see DoiDataCiteServiceBean now gets the new URL via the SystemConfig method and the DeletePIDCommand is no longer DataCite specific.

Reply:
Great. Will pass to QA.